subsys/logging/log_output.c

```diff
@@ -292,14 +295,14 @@ static u32_t hexdump_line_print(struct log_msg *msg,
 }
 
 static void hexdump_print(struct log_msg *msg,
-			  struct log_output_ctx *ctx,
+			  const struct log_output *log_output,
 			  int prefix_offset)
```

Comment:
Where are you handling description string embedded in hexdump message?

Reply:
rebased to master and now it is there.